ui-admin/src/forms/FormPreviewOptions.tsx

```diff
@@ -84,10 +93,68 @@ export const FormPreviewOptions = (props: FormPreviewOptionsProps) => {
             onChange({ ...value, locale: language?.value.languageCode ?? 'default' })
           }}/>
       </div>
-      <p className="form-text">
-        The language to use when rendering the form. If the language is not supported for this form, the default
-          language for the form will be used.
-      </p></> }
+      <div className="mt-4">
+        <div data-testid="profileInfoFields">
+          <h4 className="h6">
+          Participant profile <InfoPopup content={<p>
+          Change the values below to test how your survey appears to different participants
+          due to branching logic or dynamic texts.  The participant profile is the profile of the
+          user taking the survey.
+            </p>}/>
+          </h4>
+          <TextInput onChange={text => onChange({
+            ...value,
+            profile: {
+              ...value.profile,
+              givenName: text
+            }
+          })}
+          label={'Given name'}
+          unboldLabel={true}
+          value={value.profile?.givenName ?? ''} />
+          <TextInput onChange={text => onChange({
+            ...value,
+            profile: {
+              ...value.profile,
+              familyName: text
+            }
+          })}
+          label={'Family name'}
+          unboldLabel={true}
+          value={value.profile?.familyName ?? ''} />
+        </div>
+        <div data-testid="proxyInfoFields">
```

Comment:
Should we put this behind `config.proxyEnrollmentEnabled` check?

Reply:
hah, I considered that, but the config was only accessible about 5 components up, so it wasn't worth the drilldown.  

Related, I think there's an engineering infrastructure ticket to make the PortalContext more explicit about what it includes (e.g. config + languages) and doesn't, so that other components can work with it more easily